.github/workflows/CI.yml

```
@@ -27,17 +27,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
-        python-version: ["3.9", "3.10", "3.11"]
-        rdkit: [true, false]
-        openeye: [true, false]
+        os: [macos-latest]
```

```suggestion
        os: [macos-13]
```
Could just stick to x86 to get things green, and later on expand the matrix out?